homeassistant/components/cert_expiry/strings.json

```
@@ -20,5 +21,17 @@
       "already_configured": "This host and port combination is already configured",
       "import_failed": "Import from config failed"
     }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "data": {
```

The intention of this option is a bit unclear.
```suggestion
        "title": "Use a custom CA certificate",
        "data": {
```

---

Is there anything else to do, except the change in the string json?